internal/vendors/lenovo/lenovo.go

```diff
@@ -42,8 +42,12 @@ func InBand(board *api.Board) (hal.InBand, error) {
 
 // OutBand creates an outband connection to a Lenovo server.
 func OutBand(r *redfish.APIClient, board *api.Board, ip string, ipmiPort int, user, password string) (hal.OutBand, error) {
+	ob, err := outband.New(r, board, ip, ipmiPort, user, password)
```

Comment:
Are you sure console access works via ipmi on lenovo, i thought this would only be possible with ssh

Reply:
Indeed. But since outband is used by all vendors, the ipmitool is at least initialized (but not used by lenovo).
Nevertheless, I will provide distinct constructors to avoid this.